pkg/skaffold/build/cluster/sources/localdir.go

```diff
@@ -83,6 +88,16 @@ func (g *LocalDir) Pod(args []string) *v1.Pod {
 		Image:        g.artifact.BuildContext.LocalDir.InitImage,
 		Command:      []string{"sh", "-c", "while [ ! -f /tmp/complete ]; do sleep 1; done"},
 		VolumeMounts: []v1.VolumeMount{vm},
+		Resources: v1.ResourceRequirements{
```

Comment:
Instead of setting new defaults and parsing them here,  WDYT of reusing the [resourceRequirements](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/build/cluster/sources/sources.go#L78) function? It seems reasonable to apply the same requirements specified in the skaffold config to each container.

Reply:
I'm using the same resource requirements now 👍